content/docs/command-reference/gc.md

```diff
@@ -42,6 +42,9 @@ most cases removing data locally and in remote storage is irreversible.
 The default remote is cleaned (see `dvc config core.remote`) unless the
 `--remote` option is used.
 
+> Note that this command may pull data files or directories from remote storage
+> that are missing in DVC cache.
+
```

Comment:
Hey @santos22 thanks for this! But I'm not sure you understand the issue (#1075), granted, it's not very descriptive 😅 (which is why it's not in our good-first-issue label). You probably want to click on that Discord link and read the context. Also learn some DVC basics if you haven't, especially https://dvc.org/doc/user-guide/dvc-files-and-directories for this.

Please post any questions on the issue. Thanks

Reply:
No problem. I saw Ivan's message about adding a note to the gc docs so I figured this was all that was required. What else is missing? Looks like there was an issue with the password config for the SSH storage which was the cause for confusion when running the `gc` command. Should this be mentioned?

Reply:
> adding a note to the gc docs

Yes but what is the note about? 🙂 it's about .dir files (stored in cache and remotes) are you familiar with this mechanism? It's explained in the doc I shared. I also left notes in the original task.